src/pages/MainPage/MainPage.jsx

```diff
@@ -0,0 +1,15 @@
+import MainContainer from 'components/MainContainer/MainContainer';
+import { Link } from 'react-router-dom';
+const MainPage = () => {
+  return (
+    <MainContainer>
+      <h1>Logic Warriors</h1>
+      <p>
+        Welcome to Logic Warriors! Click <Link to={'/game'}>GAME</Link> to
+        battle!
+      </p>
+    </MainContainer>
+  );
+};
+
```

Comment:
- [ ] Great work @olhazamlynska , This is amazing 👏🏾

Reply:
Thanks